sources/features/gsub.fea

```diff
@@ -0,0 +1,752 @@
+languagesystem DFLT dflt;
+languagesystem cyrl dflt;
+languagesystem cyrl SRB ;
+languagesystem grek dflt;
+languagesystem hebr dflt;
+languagesystem latn AZE ;
+languagesystem latn CRT ;
+languagesystem latn dflt;
+languagesystem latn FIN ;
+languagesystem latn ISM ;
+languagesystem latn LSM ;
+languagesystem latn NOR ;
+languagesystem latn NSM ;
+languagesystem latn SKS ;
+languagesystem latn SSM ;
+languagesystem latn SVE ;
+languagesystem latn TRK ;
+
+# #include "locl.fea"
+
+# #include "small.fea"
+
+# #include "case.fea"
+
+# #include "contextual.fea"
+
+# #ifndef NOSMALLCAPS
+# #include "sc.fea"
+# #endif
+
+# #include "dhlig.fea"
+
+# #include "num.fea"
+
+# #include "sscv.fea"
+
+# #ifdef SANS
+# #include "liga_sans.fea"
+# #else
+# #include "liga.fea"
+# #endif
+
+feature locl {
+
+ script cyrl;
+     language SRB  exclude_dflt;
+      lookup locl_cyrillic;
+
+ script latn;
+     language AZE  exclude_dflt;
+# #ifndef SANS
+      lookup locl_undo_fi;
```

Comment:
It turns out these locl_undo_fi lines "work" only because FontForge is clever enough to implicitly "move" them below the liga entry, presumably because of the characters they reference. That's nice of FontForge but it's better to have them where they need to be, so I'm moving them into a separate feature.

Reply:
This was FontForge's ordering problem. Worked around.